backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/externalsystem/invokeExternalSystem.feature

```diff
@@ -41,3 +41,16 @@ Feature: external system invocation using metasfresh api
       | param.key | param.value |
       | OrderId   | 111         |
       | OrderNo   | 222         |
+
+
+  @from:cucumber
+  Scenario: The request is good and the external Other is invoked via the correct process
+    Given add Other external system config with identifier: otherConfigIdentifier
+      | Name | Value |
+      | Test | Test  |
```

Comment:
as we know, one "other" can have any number of parameters.
please demonstracte that it also works with >1 of those